### DISCUSSION TITLE
jsoncons  library is giving compilation error in c++20

### DISCUSSION THREAD

**Discussion by vksinghbhu2** [danielaparker/jsoncons#457] [Q&A]
Hi,

I am using jsoncons library in my visual studio C++ project. This library is giving compilation error in c++20.  is there any future plan to convert jsoncons library for c++20 compatible.

Thanks and Regards
Vijay Singh

**Comment by danielaparker** [comment]
If you experience compilation issues, please open an issue, preferably with sample code that reproduces them.

Thanks,
Daniel

**Comment by vksinghbhu2** [comment]
Code is compiled successfully at my side.  

Thanks,
Vijay
